packages/dd-trace/test/config.spec.js

```diff
@@ -514,11 +540,12 @@ describe('Config', () => {
       { name: 'traceId128BitGenerationEnabled', value: true, origin: 'env_var' },
       { name: 'traceId128BitLoggingEnabled', value: true, origin: 'env_var' },
       { name: 'spanAttributeSchema', value: 'v1', origin: 'env_var' },
-      { name: 'spanComputePeerService', value: true, origin: 'env_var' },
       { name: 'sampler.rateLimit', value: '-1', origin: 'env_var' },
       { name: 'spanRemoveIntegrationFromService', value: true, origin: 'env_var' },
-      { name: 'peerServiceMapping', value: 'c:cc,d:dd', origin: 'env_var' },
-      { name: 'tracePropagationStyle.extract', value: ['b3', 'tracecontext'], origin: 'env_var' },
+      { name: 'peerServiceMapping', value: { c: 'cc', d: 'dd' }, origin: 'env_var' },
```

Comment:
What happened here? Was the old test testing at a different level of abstraction, like before stuff was parsed?

Reply:
The old testing was not asserting any of these entries, so my guess is the assertion was always wrong.